packages/ckb-indexer/tests/transaction_collector.e2e.test.ts

```diff
@@ -84,7 +84,11 @@ test("query transactions with different queryOptions", async (t) => {
     for await (const hash of transactionCollector.collect()) {
       transactionList.push(hash);
     }
-    t.deepEqual(transactionList, queryCase.expectedResult, queryCase.desc);
+    t.deepEqual(
+      utils.deepCamelizeTransaction(utils.deepCamel(transactionList)),
```

Comment:
Can we implement the `deepCamelizeTransaction(transactionList)` rather than `utils.deepCamelizeTransaction(utils.deepCamel(transactionList))`. The `deepCamel` for `deepCamel` object keys only, and the `deepCamelizeTransaction` for both key and value like `depGroup`